docs/backuprestore-with-oadp.md

```diff
@@ -334,17 +356,53 @@ $ base64 -w0 ./credentials-velero
 W2RlZmF1bHRdCmF3c19hY2Nlc3Nfa2V5X2lkPTxBV1NfQUNDRVNTX0tFWV9JRD4KYXdzX3NlY3JldF9hY2Nlc3Nfa2V5PTxBV1NfU0VDUkVUX0FDQ0VTU19LRVk+Cg==
 ```
 
-> [!IMPORTANT]
+> [!NOTE]
 >
 > 1. The value of `spec.backupLocations[0].velero.config.profile` should match the name of the profile specified in the credentials-velero file.
 > 2. The bucket specified in `spec.backupLocations[0].velero.objectStorage.bucket` must has been created in the S3 storage backend.
```

Comment:
not touched by this PR, but in case u'd like to fix it
```suggestion
> 2. The bucket specified in `spec.backupLocations[0].velero.objectStorage.bucket` must have been created in the S3 storage backend.
```